plugins/permission-backend/src/service/router.test.ts

```diff
@@ -62,6 +66,28 @@ describe('createRouter', () => {
   let app: express.Express;
 
   beforeAll(async () => {
+    const getIdentityMock = jest.fn(
+      ({
+        request: req,
+      }: IdentityApiGetIdentityRequest): Promise<
+        BackstageIdentityResponse | undefined
+      > => {
+        const token = req.headers.authorization?.replace(/^Bearer[ ]+/, '');
+
+        if (!token) {
+          return Promise.resolve(undefined);
+        }
+
+        return Promise.resolve({
+          identity: {
+            type: 'user',
+            userEntityRef: 'test-user',
+            ownershipEntityRefs: ['blah'],
+          },
+          token,
+        });
+      },
+    ) as any;
```

Comment:
I added this any cast here because tsc wasn't passing because it assumed this function will be the other signature of the getIdentity method. Not sure how to do it properly , I wasn't really able to figure out how to mock an overloaded function in jest.